### PR TITLE
Install the Checkout Glue API

### PR DESCRIPTION
**Pull request by romanhavrylko**
## PR Description

Ticket: https://spryker.atlassian.net/browse/CC-31112

### CODE REVIEW

**Review by kkicheglovspryker — APPROVED** [review]

File: _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-checkout-glue-api.md [hunk 1]
```diff
@@ -0,0 +1,946 @@
+
+
+
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-checkout-glue-api.md:3]: do we need these spaces?

File: _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-checkout-glue-api.md [hunk 2]
```diff
@@ -0,0 +1,946 @@
...
+| Glue API: Customer Account Management   | {{page.version}} | [Install the Customer Account Management Glue API](/docs/pbc/all/identity-access-management/{{page.version}}/install-and-upgrade/install-the-customer-account-management-glue-api.html) |
+| Glue API: Payments                      | {{page.version}} | [Glue API: Payments feature integration](/docs/pbc/all/payment-service-provider/{{page.version}}/base-shop/install-and-upgrade/install-the-payments-glue-api.html)                      |
+| Glue API: Shipment                      | {{page.version}} | [Integrate the Shipment Glue API](/docs/pbc/all/carrier-management/{{page.version}}/base-shop/install-and-upgrade/install-the-shipment-glue-api.html)                                   |
+
+### 1) Install the required modules using Composer
+
+Install the required modules:
+
+```bash
+composer require spryker/checkout-rest-api:"3.8.0" spryker/order-payments-rest-api:"^1.0.0" spryker/sales-order-thresholds-rest-api:"^1.0.0" --update-with-dependencies
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-checkout-glue-api.md:27]: there is 3.10.0 already available for `spryker/checkout-rest-api`
and the patch for `spryker/order-payments-rest-api`
doe it make sense to specify the latest versions?

File: _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-checkout-glue-api.md (hunk 1 above)
Comment (romanhavrylko) [reply _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-checkout-glue-api.md:3]: This was moved from the previous version of this IG. @andriitserkovnyi, can we delete these empty lines?

**Review by DmitryLymarenko — DISMISSED** [review]

**Review by andriispryker — DISMISSED** [review]
Approved.

File: _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-checkout-glue-api.md (hunk 1 above)
Comment (andriitserkovnyi) [reply _includes/pbc/all/install-features/202403.0/install-glue-api/install-the-checkout-glue-api.md]: will do

**Pull request merged**